.changeset/popular-deers-raise.md

```diff
@@ -0,0 +1,5 @@
+---
+'openzeppelin-solidity': major
+---
+
+`Governor`: Use `bytes memory` signature instead of `r`, `s` and `v` parameters in the `castVoteBySig` and `castVoteWithReasonAndParamsBySig` functions
```

Comment:
I just thought both required their own entry. Although the main reason of this PR is to introduce 1271 compatibility, it requires this change that should be communicated on its own.

We can think of an individual changeset if you think it's needed

Reply:
My intuition is that all of the changes to `castVoteBySig` should be together in the same changeset, including also #4378 currently in `.changeset/sixty-numbers-reply.md`.

If not that, I do think we should at least merge the 2 in this PR.

Reply:
I prefer merging these two into a single one because otherwise, we'll be breaking the PR reference when processing the changesets into the changelog.

Not sure if **all** the changes should be in a single entry. But I acknowledge I'd be worried about the order in the Changelog once they get processed.